srcpkgs/puppet/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'puppet'
 pkgname=puppet
 version=6.1.0
-revision=1
+revision=2
 noarch=yes
 build_style=ruby-module
 hostmakedepends="ruby facter-devel hiera"
 makedepends="facter-devel"
-depends="ruby>=2.2 facter hiera"
+depends="ruby>=2.2 facter hiera ruby-semantic_puppet"
```

Comment:
I think the ruby>=2.2 can have the version specifier removed, we are on 2.6.0 now

Reply:
Do you want that in a seperate commit?